### PR TITLE
VirtualCards: Notify stale missing-receipt charges

### PR DESCRIPTION
**Pull request by kewitz**
Resolves https://github.com/opencollective/opencollective/issues/5269

### Card missing Receipts
Screenshot from 2022-04-06 13-56-38

### Card Suspended because of missing Receipts
Screenshot from 2022-04-06 13-56-10

### Card manually Suspended
Screenshot from 2022-04-06 15-21-39

### CODE REVIEW

File: templates/emails/collective.virtualcard.missing.receipts.hbs [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+{{> header}}
+
+<center>
+  <h1>
+    {{collective.name}}'s {{virtualCard.name}} ({{virtualCard.last4}}) card is missing receipt(s).
+  </h1>
+</center>
+
+<p>
+  Avoid having your card suspended by providing the receipt for the following charges within {{daysLeft}} days:
```
Comment (natehn) [inline templates/emails/collective.virtualcard.missing.receipts.hbs:13]: Add a sentence preceding this line: "Per your fiscal host's virtual card policy, receipts must be uploaded for virtual card expenses within 31 days."

File: templates/emails/collective.virtualcard.suspended.hbs [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    <td align="right"><a href="{{url}}">Submit Receipt</a></td>
+  </tr>
+  {{/each}}
+</table>
+{{/if}}
+
+
+<div class="info">
+  {{#if expenses}}
+  Please, submit the missing receipts and contact your fiscal host to restore your card.
```
Comment (natehn) [inline templates/emails/collective.virtualcard.suspended.hbs:31]: Replace with: "Per your fiscal host's virtual card policy, receipts must be uploaded for virtual card expenses within 31 days. Please submit all missing receipts and then contact your fiscal host to restore your card."

File: templates/emails/collective.virtualcard.suspended.hbs [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+  {{else}}
+  Please, contact your fiscal host to restore your card.
```
Comment (natehn) [inline templates/emails/collective.virtualcard.suspended.hbs:33]: Replace with: "For more information or to restore the card, please contact your fiscal host."

File: templates/emails/collective.virtualcard.missing.receipts.hbs (hunk 1 above)
Comment (kewitz) [reply templates/emails/collective.virtualcard.missing.receipts.hbs:13]: I don't think this is necessarily true for all hosts.
Any way we can make this more generic?

**Pull request merged**